### PR TITLE
chore(showcase): Add lebikini.com

### PR DESCRIPTION
**Pull request by antoinerousseau**
New website for Toulouse's most iconic concert hall.

https://lebikini.com

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -7230,7 +7230,7 @@
   featured: false
 - title: Recetas El Universo
   description: >-
-    Recipes and videos with the best of Ecuadorian cuisine. 
+    Recipes and videos with the best of Ecuadorian cuisine.
```
Comment (antoinerousseau) [inline docs/sites.yml]: this edit was made by the husky pre-commit hook linter, fyi

**Comment by wardpeet** [comment]
I'm getting an error when viewing it.

image

**Comment by antoinerousseau** [comment]
@wardpeet can you access it with www? => https://www.lebikini.com (I'm having troubles with Netlify DNS for the root domain)

**Comment by antoinerousseau** [comment]
I tried to `ping` and `curl` from multiple locations ands services and I get no issue, it responds to ping and it redirects to https://www.lebikini.com/ (`HTTP/1.1 301 Moved Permanently`).
Can you check your connection and DNS?

**Comment by muescha** [comment]
i can open it - but very slow loading ... maybe some hickups from netlify ...

**Pull request merged**